internal/cluster/cluster.go

```diff
@@ -1632,6 +1632,9 @@ func (m *Manager) GenerateAdditionalManifests(ctx context.Context, cluster *comm
 		return errors.Wrap(err, "failed to add disk encryption manifest")
 	}
 
+	if err := m.manifestsGeneratorAPI.AddNicReapply(ctx, log, cluster); err != nil {
+		return errors.Wrap(err, "failed to add nic reapply manifest")
+	}
```

Comment:
This will run on all nodes  regardless if there is an ISCSI drive.
 
does it happens on the first boot? during installation ( reboot )?

Reply:
> This will run on all nodes regardless if there is an ISCSI drive.

Yes, because we can have mixed configurations (e.g.: masters on VMs with non-iSCSI and workers on BM with iSCSI drives). That's why the script checks for an iSCSI boot volume.

> does it happens on the first boot? during installation ( reboot )?

The manifest is added to the ignition of the machine, then on reboot, the unit is create an enabled.

Reply:
@avishayt do you think it's acceptable to drop such machine config file?

Reply:
I don't know enough to say what the effect will be

Reply:
ack, maybe in would be better to be conservative, and just add this manifest only in one of the nodes is using iSCSI boot

Reply:
done!